docs/ref/templates/builtins.txt

```diff
@@ -1436,6 +1436,39 @@ then the output would be::
 
 .. templatefilter:: dictsortreversed
 
+While ``dictsort`` was designed to work with list of dictionaries, it is also
```

Comment:
This text should be above `.. templatefilter:: dictsortreversed`

I would chop the "While..." bit and say something like "dictsort can also order a list of lists..."